### ISSUE TITLE
Automatic recurring snapshots on downstream RKE2 cluster are not seen in Rancher UI

### ISSUE THREAD

**Issue by Josh-Diamond** [rancher/dashboard#8409]
**Setup**
- Rancher version: `v2.7.0` + `v2.7-76c5c80bd1069bc14c8961014bd8b559b97bf22a-head`
- Browser type & version: `Chrome`

**Describe the bug**
Downstream RKE2 (Node driver) cluster is successful in capturing recurring snapshots, but they are not displayed via Rancher UI.

**To Reproduce**
1. Fresh install of rancher `v2.7-head` or `v2.7.0`
2. Provision a downstream RKE2 (Node driver) cluster w/ automatic snapshots enabled to capture every minute - [ cron schedule: `*/1 * * * *` ]
3. Once `active`, navigate to `Cluster Management > Cluster Details > Snapshots`
4. Observe initial snapshot taken, but no recurring snaps are seen
5. Re-visit 5m later
6. Reproduced

**Result**
No recurring snapshots are seen in Rancher UI, however they are successfully captured and available on etcd node

**Expected Result**
All captured snapshots should be available via Rancher UI

**Additional context**

Rancher Logs:
```logs
2023/03/10 23:00:39 [INFO] [snapshotbackpopulate] rkecluster fleet-default/jkeslar-rke2-snap: processing configmap kube-system/rke2-etcd-snapshots
2023/03/10 23:00:39 [INFO] [snapshotbackpopulate] rkecluster fleet-default/jkeslar-rke2-snap: processing configmap kube-system/rke2-etcd-snapshots
2023/03/10 23:00:39 [INFO] [snapshotbackpopulate] rkecluster fleet-default/jkeslar-rke2-snap: processing configmap kube-system/rke2-etcd-snapshots
```

**Comment by mantis-toboggan-md** [comment]
I don't think this is a UI issue. I've followed the repro steps above and I also see unexpected behavior on the snapshot tab: I see no snapshots, then, <10 minutes later, 6 appear with the same 'age'. The UI is looking for `rke.cattle.io.etcdsnapshots` in the local cluster...I've watched the relevant websocket messages as well as checked what GET requests to `<rancher url>/v1/rke.cattle.io.etcdsnapshots` return and the UI is displaying the information available there correctly.
